core/src/CMakeLists.txt

```diff
@@ -65,13 +61,19 @@ set(_srcs
 
 set(_private_headers
   util/usFrameworkPrivate.h
+  util/usLDAPExpr_p.h
+  util/usProperties_p.h
   util/usThreads_p.h
   util/usUtils_p.h
   util/usWaitCondition_p.h
 
   service/usServiceHooks_p.h
+  service/usServiceListenerEntry_p.h
```

Comment:
These changes were made, because some of the files were moved; and made additional changes as I realized that some of the files were missing from this list.